src/eq_fault_geom/geomio/cfm_shp_to_vtk.py

```diff
@@ -0,0 +1,214 @@
+from pathlib2 import Path
+import string
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import LineString
+import numpy as np
+import meshio
+# import pdb
+# pdb.set_trace()
+
+# Epsilon value.
+eps = 5000.0
+
+# Cell type.
+cell_type = 'triangle'
+# cell_type = 'quad'
+
+# Output directory and file suffix.
```

Comment:
These lines (#18-24) belong either inside the `if __main__` block or wrapped in a function ... 
Here they'll try to execute and fail when the modeul ios imported